cypress/e2e/brewery/Router-RedirectionFromDisabledView.cy.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import { DatasetManager, InstanceVisualization, Login } from '../../commons/actions';
+import { GENERIC_SELECTORS } from '../../commons/constants/generic/IdConstants';
+import { stub } from '../../commons/services/stubbing';
+import { routeUtils as route } from '../../commons/utils';
+import {
+  WORKSPACE_LIST_WITHOUT_DASHBOARDS,
+  WORKSPACE_WITHOUT_DASHBOARDS,
+} from '../../fixtures/stubbing/ScenarioViewDashboard/workspace';
+import { DEFAULT_SCENARIOS_LIST } from '../../fixtures/stubbing/default';
+
+const scenarioId = DEFAULT_SCENARIOS_LIST[0].id;
+
+describe('redirection from disabled view', () => {
+  before(() => {
+    stub.start();
+    stub.setWorkspaces(WORKSPACE_LIST_WITHOUT_DASHBOARDS);
+  });
+
+  beforeEach(() => {
+    Login.login({
+      url: `/${WORKSPACE_WITHOUT_DASHBOARDS.id}`,
+      workspaceId: WORKSPACE_WITHOUT_DASHBOARDS.id,
+      isPowerBiEnabled: false,
+    });
+  });
+
+  after(() => {
+    stub.stop();
+  });
+  it('can redirect from disabled Instance view to scenario view', () => {
+    route.browse({
+      url: `/${WORKSPACE_WITHOUT_DASHBOARDS.id}`,
```

Comment:
Shouldn't we try to access the URL of the specific views (instance view, dataset manager, dashboards view) in this test? The test does it for the dashboards view, but not for the instance view and dataset manager.

Reply:
too much refactoring can kill the test :woman_facepalming: